tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -217,7 +217,7 @@ public class Class1 {
 			};
 			proj.SetAndroidSupportedAbis ("armeabi-v7a", "arm64-v8a", "x86", "x86_64");
 			proj.SetProperty (proj.ReleaseProperties, "MonoSymbolArchive", "True");
-			proj.MainActivity = proj.DefaultMainActivity.Replace ("//${AFTER_ONCREATE}",
+			proj.MainActivity = proj.DefaultMainActivity.Replace ("//${AFTER_FORMS_INIT}",
```

Comment:
I found this test was not using a Forms-based `MainActivity.cs` before, but it was a Xamarin.Forms app! So we need to use `${AFTER_FORMS_INIT}` now and the line number changed.